pkg/tfbridge/provider_test.go

```diff
@@ -331,5 +369,15 @@ func TestProviderPreviewV2(t *testing.T) {
 			Schema: &ResourceInfo{Tok: "ExampleResource"},
 		},
 	}
+
 	testProviderPreview(t, provider)
+
+	failures := testCheckFailures(t, provider)
+	sort.SliceStable(failures, func(i, j int) bool { return failures[i].Reason < failures[j].Reason })
+	assert.Equal(t, "ConflictsWith: \"conflicting_property\": conflicts with conflicting_property2", failures[0].Reason)
+	assert.Equal(t, "", failures[0].Property)
+	assert.Equal(t, "ConflictsWith: \"conflicting_property2\": conflicts with conflicting_property", failures[1].Reason)
+	assert.Equal(t, "", failures[1].Property)
+	assert.Equal(t, "Required attribute is not set", failures[2].Reason)
```

Comment:
It's unfortunate that the error here still does not include the path of the required attribute.  I believe Terraform relies on `AttributePath` in the diagnostic to communicate this - but not clear if/how we should be surfacing that ourselves into our errors.

Reply:
> I believe Terraform relies on AttributePath in the diagnostic to communicate this 

Yes, I think that you're right. I do believe that we may be able to translate an `AttributePath` into a Pulumi property path--I think that all we need is the relevant TF schema and Pulumi annotations.

Reply:
Opened https://github.com/pulumi/pulumi-terraform-bridge/issues/311 to track this as a follow-up.